src/core/public/chrome/chrome_service.test.ts

```diff
@@ -82,6 +76,16 @@ async function start({
   };
 }
 
+beforeAll(() => {
+  // @ts-ignore
+  delete window.localStorage;
+  (window as any).localStorage = {
```

Comment:
I see this is done in multiple files, but just to understand, why are the added `delete` statements required for this bump?

Reply:
Good point @pgayvallet. So jest bumped used version of jsdom from v11 to v15
https://github.com/facebook/jest/releases/tag/v25.1.0
and the new version of jsdom doesn't allow to easily mock `location` properties as the old one did :( 
https://remarkablemark.org/blog/2018/11/17/mock-window-location/

Reply:
So the `window.location` (and others) properties are now read-only. Wow, that's a shame.

Just to be sure, is this also needed when overriding the whole `window.localStorage` object (`window.localStorage = ...`) and not sub-properties (`window.localStorage.something = ...`). 

Would the `window.localStorage = ...` statement really fails here without the previous `delete` statement (I'm a little surprised that read-only access would forbid to override the prop but allow to delete it)

Reply:
can we add this as a comment in the source?  mostly on first glance it looks a little strange